### PR TITLE
Feat: fix lint warnings and formatting

### PR DESCRIPTION
**Pull request by cyber-eternal**
Documentation content update for CONTRIBUTING.md TECHNICAL-SETUP.md and versions.md

This PR Feat: fix lint warnings and formatting

Change scope
The changes in this Pull Request include (please tick all that apply):

* [x] Small language/grammar fixes
* [x] Small content fixes 
* [ ] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

**Comment by cyber-eternal** [comment]
Dear [xavierlepretre](https://github.com/xavierlepretre), I have opened a new PR since there was something wrong with my forked repo and I couldn't commit to it.  RE: your comment `I am not sure why to change the indentation from 4 spaces to 3.`. It was fixed here. Thanks!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -103,18 +97,18 @@ Since the deploy preview does not work on Draft PRs, follow these steps to previ
...
+   $ npm run serve
+   ```
 
-  A successful client compile looks like: `> VuePress dev server listening at http://localhost:8080/ ✔ Client Compiled successfully in 280.71ms success [12:06:28] Build 03d41f finished in 283 ms! ( http://localhost:8080/ )`
+    A successful client compile looks like: `> VuePress dev server listening at http://localhost:8080/ ✔ Client Compiled successfully in 280.71ms success [12:06:28] Build 03d41f finished in 283 ms! ( http://localhost:8080/ )`
 
 5. You can now view the tutorial build on a local web browser. Isn't this fun?
 
-    **Tip:** On a mac, press the command key and click `http://localhost:8080/` for quick access.
+   **Tip:** On a mac, press the command key and click `http://localhost:8080/` for quick access.
```
Comment (xavierlepretre) [inline CONTRIBUTING.md:114]:
```suggestion
    **Tip:** On a mac, press the command key and click `http://localhost:8080/` for quick access.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -103,18 +97,18 @@ Since the deploy preview does not work on Draft PRs, follow these steps to previ
 
 3. For each branch you work in, install the npm packages for the SDK tutorials:
 
-    ```sh
-    $ npm install
-    ```
+   ```sh
```
Comment (xavierlepretre) [inline CONTRIBUTING.md:100]:
```suggestion
    ```sh
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -103,18 +97,18 @@ Since the deploy preview does not work on Draft PRs, follow these steps to previ
...
+   $ npm install
```
Comment (xavierlepretre) [inline CONTRIBUTING.md:101]:
```suggestion
    $ npm install
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -103,18 +97,18 @@ Since the deploy preview does not work on Draft PRs, follow these steps to previ
...
+   ```
```
Comment (xavierlepretre) [inline CONTRIBUTING.md:102]:
```suggestion
    ```
```

File: TECHNICAL-SETUP.md [hunk 5]
```diff
@@ -237,7 +237,7 @@ It is recommended that you master your terminal to be happy.
 
 ### iTerm2 Terminal Emulator
 
-On MacOS, install the [iTerm2](https://iterm2.com/) OSS terminal emulator as a replacement for the default Terminal app. Installing iTerm2 as a replacement for Terminal provides an updated version of the Bash shell that supports useful features like programmable completion.
+On MacOS, install the [iTerm2](https://iterm2.com/) OSS terminal emulator as a replacement for the default Terminal app. Installing iTerm2 as a replacement for Terminal provides an updated version of the sh shell that supports useful features like programmable completion.
```
Comment (xavierlepretre) [inline TECHNICAL-SETUP.md:240]: Was the change from Bash to sh also prompted by the linter?

Comment (cyber-eternal) [reply TECHNICAL-SETUP.md]: Nope. It was a mistake. Fixed

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -87,11 +87,11 @@ Since the deploy preview does not work on Draft PRs, follow these steps to previ
 
 1. If you have not already, clone the tutorials repo to your local machine and change to that directory. For example:
 
-   ```sh
-   cd ~/github
-   git clone --recursive https://github.com/cosmos/sdk-tutorials
-   cd sdk-tutorials
-   ```
+    ```bash
```
Comment (xavierlepretre) [inline CONTRIBUTING.md]: Actually, the `sh` here is ok. We use it everywhere else in our content.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -109,11 +103,11 @@ Since the deploy preview does not work on Draft PRs, follow these steps to previ
 
 4. Start the local instance of the tutorial build:
 
-    ```sh
-    $ npm run serve
-    ```
+   ```sh
+   $ npm run serve
+   ```
```
Comment (xavierlepretre) [inline CONTRIBUTING.md:108]: I am confused why there are still 3 spaces on this indentation. I thought your commits reverted that.

Comment (cyber-eternal) [reply CONTRIBUTING.md]: Done!

File: CONTRIBUTING.md (hunk 6 above)
Comment (cyber-eternal) [reply CONTRIBUTING.md]: updated to sh

**Review by xavierlepretre — APPROVED** [review]

**Comment by coldice** [comment]
@cyber-eternal thank you for your contribution.

**Pull request merged**